src/pages/about/_join-us.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import styled from 'styled-components'
+import device from 'themes/device'
+import { Header, Text } from 'components/elements'
+import { localize, LocalizedLink } from 'components/localization'
+
+const JoinUsContainer = styled.section`
+    display: flex;
+    justify-content: flex-end;
+    width: 100%;
+    padding: 4rem 9.7rem;
+    margin: auto;
+    align-items: right;
+    background-image: linear-gradient(to left, #232232, #223122 81%, rgba(0, 0, 0, 0.26) 50%);
```

Comment:
we can use rgba for this instead of hex

Reply:
This section was removed. Probably they will change it. I'll consider it in new implementation. Thanks.